seatunnel-core/seatunnel-core-base/src/main/java/org/apache/seatunnel/config/command/CommandFlinkArgs.java

```diff
@@ -42,4 +44,7 @@ public boolean isTestConfig() {
         return testConfig;
     }
 
+    public List<String> getVariables() {
```

Comment:
It seems that we never use `getVariables()` in main logic.

Reply:
UT used.

Reply:
> It seems that we never use `getVariables()` in main logic.

UT used.